test/suite/test_count01.py

```diff
@@ -31,16 +31,55 @@
 #
 
 import wiredtiger, wttest
+from wtdataset import SimpleDataSet
+from wtscenario import make_scenarios
 
 class test_count01(wttest.WiredTigerTestCase):
     tablename = 'test_count01'
-    uri = 'table:' + tablename
+    tablename2 = 'test'
+
+    keyfmt = [
+        ('integer-row', dict(keyfmt='i')),
+        ('column', dict(keyfmt='r')),
```

Comment:
Should we increase test for more types of tables? Fixed length column store and column groups if we support it.

Reply:
I have added a fixed-length column store scenario. I'm not sure if we can test the column groups scenario, given that it won't be accurate all the time.